client/src/app/site/pages/meetings/view-models/view-user.ts

```diff
@@ -233,6 +234,10 @@ export class ViewUser extends BaseViewModel<User> /* implements Searchable */ {
         return this.getMeetingUser(meetingId)?.structure_level_ids;
     }
 
+    public structure_level(meetingId?: Id): ViewStructureLevel[] {
```

Comment:
`structure_levels` would be better as this contains an array of structure levels.